api/jobs.yaml

```diff
@@ -116,6 +131,40 @@ paths:
       tags:
         - Jobs
 definitions:
+  AuthenticationCapability:
+    description: Specification of the backend's authentication capabilities
+    properties:
+      is_required:
```

Comment:
Actually, we do use camelCase elsewhere, should use that here as well.

e.g. parentId

Reply:
Good point, didn't notice that. Switched all snake_case to camelCase